conf/test.conf

```diff
@@ -0,0 +1,5 @@
+include "application.conf"
```

Comment:
Does not looks necessary since it is using basically the same values in `application.conf`.

Reply:
@marcospereira The reason for including an application.conf some configurations are common for all environments.

Generally application.conf we used for dev environment. For running unit test we have to change some configurations(for eg:  database credentials ) and some reuse from the application.conf.

Reply:
@satendrakumar, yep, we all agree that's a good thing to do. I think Marcos point is that if we are using that sample to also educate people on how to tweak configurations for different environments, then it's better to leave a small comment explaining the technique.

Reply:
@renatocaval thanks for the explanation. Done.